README.md

```diff
@@ -130,3 +133,27 @@ stream.subscribe_address(sushi_swap_address, txn_handler)
 # Start the websocket connection and start receiving events!
 stream.connect()
 ```
+
+### Connecting to Binance Smart Chain
+
+```python
+from blocknative.stream import Stream
+import json
+
+# Initialize the stream - specify network_id `56` to connect to bsc main
+stream = Stream('<API_KEY>', network_id=56)
```

Comment:
It might be a good idea to define '56'. is BSC_NETWORK_ID as this is a manifest constant.